zipkin-server/pom.xml

```diff
@@ -18,11 +18,14 @@
   <modelVersion>4.0.0</modelVersion>
 
   <parent>
-    <groupId>io.zipkin.java</groupId>
+    <groupId>io.zipkin.zipkin2</groupId>
     <artifactId>parent</artifactId>
     <version>2.10.2-SNAPSHOT</version>
   </parent>
 
+  <!-- Group ID left alone due to folks downloading it with curl from maven central.
+       TODO: setup a maven redirect -->
```

Comment:
An issue should probably be created for this before merge

Reply:
https://github.com/openzipkin/zipkin/issues/2132 thanks for the reminder!